doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -468,11 +631,24 @@ Response code:
 
 + Response 204
 
+### Update existing entity attributes [PATCH /v2/entities/type/{entityType}/id/{entityId}]
+
+The request payload is an object representing the attributes to update. The object follows
+the JSON entity representation format (described in a section above), except that `id` and `type`
+are not allowed.
```

Comment:
The introduction to PUT/PATCH/POST is very similar, but not exaclty the same block. There is some repetition but I haven't find a best way of describing this...